python/taichi/lang/__init__.py

```diff
@@ -309,11 +323,11 @@ def adaptive_arch_select(arch):
     supported = supported_archs()
```

Comment:
```suggestion
    import taichi as ti
    supported = supported_archs()
```
otherwise the `ti.info` below will crash.